packages/client/src/Rest.ts

```diff
@@ -30,6 +32,10 @@ function serialize(body: any): string | undefined {
     return typeof body === 'string' ? body : JSON.stringify(body)
 }
 
+export const createQueryString = (query: Record<string, any>) => {
+    return new URLSearchParams(omitBy(query, isNil)).toString()
```

Comment:
Using `lodash/omitBy` here likely adds bloat to the browser bundle, and the vanilla version is straightforward enough:

```js
Object.fromEntries(Object.entries(query).filter(([k, v]) => v != null))
```